SU-SWS/stanford_paragraph_types
# EARTH-299 Double filmstrip paragraph type

# READY FOR REVIEW

# Summary
- Double filmstrip paragraph type
- Film card paragraph type used in double filmstrip (Should this be renamed)
- Changed all paragraph forms to closed like complex page

# Needed By (Date)
- End of Sprint

# Steps to Test

1. Checkout this branch & same branch in [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/35) & `EARTH-296-callout-filmstrip` branch of [stanford_components](https://github.com/SU-SWS/stanford_components/pull/69) (If it hasn't already been merged)
2. config import & cache clear
3. create complex page with "Double Filmstrip paragraph"
4. Verify it looks normal.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/69
- https://github.com/stanford-earth/SE3_BLT/pull/35

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
